services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
```

Comment:
Consider converting `movie_show_time` from a string to a `datetime` object before passing it to `MovieSession.objects.create`. This ensures that the `show_time` field is stored correctly as a `DateTimeField`.